osu.Game.Rulesets.Catch.Tests/TestCaseCatcherArea.cs

```diff
@@ -26,7 +26,6 @@ public class TestCaseCatcherArea : OsuTestCase
         public TestCaseCatcherArea()
         {
             AddSliderStep<float>("CircleSize", 0, 8, 5, createCatcher);
-            AddToggleStep("Hyperdash", t => catcherArea.ToggleHyperDash(t));
```

Comment:
I think it is unused? I'm not sure if it fully makes sense to toggle hyper-dash state without specifying speed / targets. So I removed this for now.